maven-java/kalix-maven-archetype-event-sourced-entity/src/main/resources/archetype-resources/pom.xml

```diff
@@ -132,7 +132,7 @@
               <name>${D}{dockerImage}:%l</name>
               <build>
                 <!-- Base Docker image which contains jre-->
-                <from>docker.io/library/adoptopenjdk:${D}{jdk.target}-jre-hotspot</from>
```

Comment:
We were previously using the jdk version here, but it should pin to fixed version.